From 08236f0ecf2505ffcc9f747aa7dee00565a1d95f Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Sat, 28 Jan 2023 20:24:32 +0800
Subject: [PATCH] Revert "Linux 5.10.165"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 179624a57b78c02de833370b7bdf0b0f4a27ca31.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5fbff8603f44..68fd49d8d436 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 165
+SUBLEVEL = 164
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

